doc/api_ref/pubkey.rst

```diff
@@ -881,9 +1039,14 @@ signature is validated.
 
 .. _emsa:
 
-Available signature padding schemes
+RSA signature padding schemes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+These signature padding mechanisms are specific to RSA; no other public
+key algorithms included in Botan make use of then. For historical reasons,
+many different padding schemes have been defined for RSA over the years.
+The most common are PSS and the (now obsolete) PKCS1v15.
+
```

Comment:
Thanks! I think this is really crucial information for crypto newcomers!

Reply:
The situation was a lot more confused in earlier releases

[Pre 2.0] Rabin-Williams and Nyberg-Rueppel could use the same padding schemes as RSA

"EMSA1" (aka the plain hash with possible truncation) was a real padding scheme that was used with *DSA and could be used with RSA as well.

Now things are pretty clearly bifurcated to "RSA specific" and "everyone else"